### DISCUSSION TITLE
GPL 2.0 licence

### DISCUSSION THREAD

**Discussion by Adeline-Toader** [PyMySQL/mysqlclient#721] [Q&A]
Hello
Is the licence of the connector GPL 2.0 only or GPL 2.0 or later? I see in the header of this file: https://github.com/PyMySQL/mysqlclient/blob/a958c5fa45d3833e9fbf31ce0c4a461782929067/src/MySQLdb/_mysql.c#L5 the reference to GPL 2.0 or later but I cannot say if it applies only to that file or to the entire project. Your clarification is more than welcome! Thank you!

**Comment by mschoettle** [comment]
This has recently been clarified in #722 after my comment here: https://github.com/PyMySQL/mysqlclient/issues/425#issuecomment-2327324722.

It is `GPL-2.0-or-later`.
